.github/workflows/dependabot-automation.yml

```diff
@@ -0,0 +1,22 @@
+name: Dependabot automation
+
+on: pull_request
+
+permissions:
+  pull-requests: write
+  contents: write
+
+jobs:
+  merge-auspice-bump:
+    if: github.event.pull_request.user.login == 'dependabot[bot]'
+    runs-on: ubuntu-latest
+    steps:
+      - id: metadata
+        uses: dependabot/fetch-metadata@v1
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+      - if: contains(steps.metadata.outputs.dependency-names, 'auspice')
```

Comment:
This output is [described as](https://github.com/dependabot/fetch-metadata):

> A comma-separated list of the package names updated by the PR.

which means that `contains()` will be a naive match and may produce false positives.

How about instead using `steps.metadata.outputs.updated-dependencies-json` and looking for an entry with `dependencyName == 'auspice'`?

Relatedly, we might want to limit auto-merges to cases where `updateType != 'version-update:semver-major'` (or similar)? (This is a bit far forward looking, but still…)

Reply:
It'd be something like:

```yaml
      - if: contains(fromJSON(steps.metadata.outputs.updated-dependencies-json).*.dependencyName, 'auspice')
```

Reply:
Good suggestions! I'll try them out on my fork repo.

Reply:
Tested, works well:

- https://github.com/victorlin/nextstrain.org/pull/5
- https://github.com/victorlin/nextstrain.org/pull/6